### PR TITLE
Scaffold

### PR DESCRIPTION
**Pull request by AKidd95**
#### Description
Changes proposed in this pull request:
* Scaffold 5.2 rails app

#### What I have learned working on this feature: 
* How to install and use a beta version of rails

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+ruby '2.4.1'
+
+gem 'rails', '~> 5.2.0.rc2'
+gem 'pg', '>= 0.18', '< 2.0'
+gem 'puma', '~> 3.11'
+gem 'bootsnap', '>= 1.1.0', require: false
+
+group :development, :test do
+  gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
```
Comment (tochman) [inline Gemfile]: Byebug? Why not pry?

File: Gemfile [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+end
+
+group :development do
+  gem 'listen', '>= 3.0.5', '< 3.2'
+  gem 'spring'
+  gem 'spring-watcher-listen', '~> 2.0.0'
+end
+
+gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
```
Comment (tochman) [inline Gemfile]: This one is not needed

File: Gemfile (hunk 1 above)
Comment (AKidd95) [reply Gemfile]: was planning on adding this during the testing environment setup. But I will add it to this initial scaffold

**Review by isabellelii — APPROVED** [review]
Looks good!

**Review by NellaMori — APPROVED** [review]
Looks good to me!

**Pull request merged**